src/js/app.js

```diff
@@ -139,7 +139,7 @@ const Application = App.extend({
   onStart(options, currentUser, { default: AppFrameApp }) {
     datadogRum.setUser(currentUser.pick('id', 'name', 'email'));
 
-    if (!currentUser.isActive() || !currentUser.get('enabled')) {
+    if (!currentUser.hasTeam() || !currentUser.get('enabled')) {
```

Comment:
I added this which is the bug from last week.  Specifically the biggest change is that if the user has no groups the `group` filter is not run on the worklist/schedule endpoints.